Formula/httpd.rb

```diff
@@ -46,6 +223,12 @@ def install
                           "--with-pcre=#{Formula["pcre"].opt_prefix}"
     system "make", "install"
 
+    # suexec does not install without root
+    bin.install "support/suexec"
+
+    index_file = var/"www/htdocs/index.html"
+    rm index_file if index_file.exist? && identical?(index_file, "docs/docroot/index.html")
+
     # remove non-executable files in bin dir (for brew audit)
     rm bin/"envvars"
     rm bin/"envvars-std"
```

Comment:
Below here: let's do something like:

```patch
     # avoid using Cellar paths
     inreplace %W[
-      #{lib}/httpd/build/config_vars.mk
       #{include}/httpd/ap_config_layout.h
+      #{lib}/httpd/build/config_vars.mk
     ] do |s|
       s.gsub! "#{lib}/httpd/modules", "#{HOMEBREW_PREFIX}/lib/httpd/modules"
+    end
+    inreplace %W[
+      #{include}/httpd/ap_config_auto.h
+      #{include}/httpd/ap_config_layout.h
+      #{lib}/httpd/build/config_vars.mk
+      #{lib}/httpd/build/config.nice
+    ] do |s|
       s.gsub! prefix, opt_prefix
     end
```

to cover all installed paths.

Reply:
And in general after your changes: `grep "/usr/local/Cellar" /usr/local/opt/httpd /usr/local/etc/httpd` and let's try to eliminate all of them.